pkg/registry/populator.go

```diff
@@ -341,7 +341,7 @@ func decodeFile(path string, into interface{}) error {
 	return decoder.Decode(into)
 }
 
-func parseDependenciesFile(path string, depFile *DependenciesFile) error {
+func ParseDependenciesFile(path string, depFile *DependenciesFile) error {
```

Comment:
Why are we changing this from a non-exported function to an exported function? I don't see it being called anywhere else outside this package.

Reply:
Well, i called it in synthesizing the CSV. This is the only way to parse the dependency file.

Reply:
Gotcha.